mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/FieldInfoMirror.cs

```diff
@@ -200,6 +200,20 @@ public Boolean IsNotSerialized
 					res.Add (attr);
 			return res.ToArray ();
 		}
+
+		public string FullName {
+			get {
```

Comment:
Good catch, I'll reword the commit. I'm basically making it symmetric with MethodInfoMirror.